docs/pages/Box.md

```diff
@@ -5,23 +5,23 @@ Use the `Box` component to control width, margin, padding, and color.
 50% width
 
 ```.jsx
-<Box width={1/2} bg='blue'>
+<Box width={1/2} color='primary.base'>
```

Comment:
😍

Reply:
If you're using `base` for a prop it's best to leave it as `primary` and this is true for all colors since if the component have a hover state the DS will take care of that for you and by default use the `dark` color. However, this is only true for working with props, getPaletteColor requires you to be specific.

Reply:
Not important for box and it's fine to leave the documentation the way it is here but I thought I'd bring it up as I taught someone that the other day.

Reply:
@pcln-james some more documentation on `getPaletteColor` https://priceline.github.io/design-system/utilities/getPaletteColor#palette-color-driven-from-color-prop